src/stan/lang/function_signatures.h

```diff
@@ -768,15 +768,20 @@ add("multi_gp_cholesky_lpdf", expr_type(double_type()), expr_type(matrix_type())
     }
   }
 }
-for (const auto& t : all_vector_types) {
-  add("multi_normal_rng", rng_return_type<double_type>(t), t, expr_type(matrix_type()));
- }
-for (const auto& t : all_vector_types) {
-  add("multi_normal_cholesky_rng", rng_return_type<double_type>(t), t, expr_type(matrix_type()));
- }
-for (const auto& t : all_vector_types) {
-  add("multi_student_t_rng", rng_return_type<double_type>(t), expr_type(double_type()), t, expr_type(matrix_type()));
- }
+
+add("multi_normal_rng", expr_type(vector_type()), expr_type(vector_type()), expr_type(matrix_type()));
```

Comment:
Is vectorization not implemented for arrays of covariance matrices/Cholesky factors?  If that was intentional because it's not implemented in the math library, please open a math library issue to complete the implementations with arrays of covariance matrices.